api/v1alpha1/manifest_types.go

```diff
@@ -172,14 +89,26 @@ const (
 // Manifest is the Schema for the manifests API.
 type Manifest struct {
 	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
```

Comment:
this is not required as `metav1.ObjectMeta` should never be empty or?

Reply:
I mainly updated these fields to conform with Kubernetes defaults, see Deployment: https://github.com/kubernetes/api/blob/master/apps/v1/types.go#L362